modules/cbMap/processmap/AdvancedSearch.php

```diff
@@ -0,0 +1,81 @@
+<?php
+ /*************************************************************************************************
+ * Copyright 2021 JPL TSolucio, S.L. -- This file is a part of TSOLUCIO coreBOS Customizations.
+ * Licensed under the vtiger CRM Public License Version 1.1 (the "License"); you may not use this
+ * file except in compliance with the License. You can redistribute it and/or modify it
+ * under the terms of the License. JPL TSolucio, S.L. reserves all rights not expressly
+ * granted by the License. coreBOS distributed by JPL TSolucio S.L. is distributed in
+ * the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. Unless required by
+ * applicable law or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT ANY WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License. You may obtain a copy of the License
+ * at <http://corebos.org/documentation/doku.php?id=en:devel:vpl11>
+ *************************************************************************************************
+ *  Module       : Business Mappings:: AdvancedSearch
+ *  Author       : JPL TSolucio, S. L.
+ *************************************************************************************************
+ * The accepted format is:
+    <map>
+        <modulename>cbCalendar</modulename>
+        <group>
+            <conditions>
+                <condition>
+                    <fieldname>Subject</fieldname>
+                    <operator>eq</operator>
+                </condition>
+                <condition>
+                    <join>and</join>
+                    <fieldname>Subject</fieldname>
+                    <operator>eq</operator>
+                </condition>
+            </conditions>
+        </group>
+        <group>
+            <groupjoin>and</groupjoin>
+            <conditions>
+                <condition>
+                    <fieldname>Subject</fieldname>
+                    <operator>eq</operator>
+                </condition>
+            </conditions>
+        </group>
+    </map>
+ *************************************************************************************************/
+
+class AdvancedSearch extends processcbMap {
+
+	public function processMap($module) {
+		$parsedxml = $this->getXMLContent();
+        $module = (string)$module[0];
```

Comment:
formatting is incorrect here: indentation

Reply:
That is strange. In my code it looks fine
![image](https://user-images.githubusercontent.com/59079190/200774594-ba1a82df-5ced-48cf-8fcf-f73b8ab7c03d.png)

Reply:
I edited it from github